knpu/custom-validation-property-path.rst

```diff
@@ -36,7 +36,7 @@ that looks like this (with some extras, like getter and setter methods)::
          */
         private $id;
 
-        /** @ORM\Column(name="name") */
+        /** @ORM\Column(name="name", type="string", length=255) */
```

Comment:
Ah, it actually defaults to these! But I agree that it's more clear without them :+1: